src/lando/ui/jinja2/partials/layout.html

```diff
@@ -1,41 +1,49 @@
+{% if request %}
+    {% set user_is_authenticated = request.user.is_authenticated %}
+    {% set user_has_phabricator_token = request.user.profile and request.user.profile.phabricator_api_key %}
+{% else %}
+    {% set user_is_authenticated = False %}
+    {% set user_has_phabricator_token = False %}
+    {% set request = None %}
+{% endif %}
```

Comment:
Could these be implemented as template helper functions? Seems like we will need to check for auth/tokens etc in other parts of the code.

Also, this change seems like it belongs in a different PR.